packages/grafana-ui/src/components/Logs/LogDetails.tsx

```diff
@@ -102,72 +125,92 @@ class UnThemedLogDetails extends PureComponent<Props> {
   };
 
   render() {
-    const { row, theme, onClickFilterOutLabel, onClickFilterLabel, getRows } = this.props;
+    const {
+      row,
+      theme,
+      onClickFilterOutLabel,
+      onClickFilterLabel,
+      getRows,
+      showDuplicates,
+      hasHoverBackground,
+      onChangeHoverBackground,
+    } = this.props;
     const style = getLogRowStyles(theme, row.logLevel);
+    const styles = getStyles(theme);
     const labels = row.labels ? row.labels : {};
     const labelsAvailable = Object.keys(labels).length > 0;
     const fields = this.getAllFields(row);
     const parsedFieldsAvailable = fields && fields.length > 0;
 
     return (
-      <div className={style.logDetailsContainer}>
-        <table className={style.logDetailsTable}>
-          <tbody>
-            {labelsAvailable && (
-              <tr>
-                <td colSpan={5} className={style.logDetailsHeading} aria-label="Log Labels">
-                  Log Labels:
-                </td>
-              </tr>
-            )}
-            {Object.keys(labels).map(key => {
-              const value = labels[key];
-              return (
-                <LogDetailsRow
-                  key={`${key}=${value}`}
-                  parsedKey={key}
-                  parsedValue={value}
-                  isLabel={true}
-                  getStats={() => calculateLogsLabelStats(getRows(), key)}
-                  onClickFilterOutLabel={onClickFilterOutLabel}
-                  onClickFilterLabel={onClickFilterLabel}
-                />
-              );
-            })}
+      <tr
+        className={cx({ [styles.hoverBackground]: hasHoverBackground })}
+        onMouseEnter={() => onChangeHoverBackground(true)}
+        onMouseLeave={() => onChangeHoverBackground(false)}
+      >
+        {showDuplicates && <td />}
+        <td className={cx(style.logsRowLevel, styles.logsRowLevelDetails)} />
+        <td colSpan={10}>
```

Comment:
Not sure I understand why do we have colSpan = 10/5? I thought we have like 3-4 columns and so if we need some column to span multiple column we would just use the number of columns to span. Am I missing something here?

Reply:
Yeah, so regarding 
**10:**
We have 4 columns, but I was just trying to ensure, that the table will go all the way to the end, even if we add additional columns, so we don't have to think about it later. But I understand that this is confusing and probably it would be better, if this was changed, when the number of rows change. 
**5:**
This is related to title-rows in log-details-table, which has 5 columns:
![image](https://user-images.githubusercontent.com/30407135/70804589-267a6e80-1db7-11ea-8ebd-6c20bc8a0dbd.png)

Reply:
Ah ok so if we add another column we will still span the whole table, right? Yeah that is good point, would be great to have something like `colSpan="100%"` so it is always full width.

Reply:
yeah, I know.